server/jetstream.go

```diff
@@ -54,6 +54,7 @@ type JetStreamStats struct {
 	ReservedMemory uint64            `json:"reserved_memory"`
 	ReservedStore  uint64            `json:"reserved_storage"`
 	Accounts       int               `json:"accounts"`
+	RaftAssets     int               `json:"raft_assets"`
```

Comment:
I think Ben was already tracking this so need to alert him. Also, might move away from raft hence why I called them HA vs Raft, so let's put that back.

Reply:
reverted